tutorials/manage-instances-with-serverless-framework/index.mdx

```diff
@@ -0,0 +1,147 @@
+---
+meta:
+  title: Managing Scaleway Instances with the serverless framework
+  description: Discover how to manage your Scaleway Instances with the serverless framework and python
+content:
+  h1:  Managing compute Instances with the serverless framework
+  paragraph: Discover how to manage your Scaleway Instances with the serverless framework and python
+tags: serverless instances python
+categories:
+  - compute
+  - serverless
+dates:
+  validation: 2021-12-21
+  posted: 2021-12-21
+---
+
+In this tutorial you will discover some examples of Instance automation using Python and the serverless framework:
+
+
+<Message type="requirement">
+
+  - You have an account and are logged into the [Scaleway Console](https://console.scaleway.com)
+  - You have [generated an API key](/console/my-project/how-to/generate-api-key/)
+  - You have [pip](https://pypi.org/project/pip/) installed on your machine
```

Comment:
Maybe also add requirements to install npm and serverless, as described in /compute/functions/api-cli/fun-uploading-with-serverless-framework/ ?